src/vehicle.cpp

```diff
@@ -41,6 +41,13 @@
 #include <numeric>
 #include <algorithm>
 
+/**
+ *  Constant use to convert between wheel and engine rpm
+ *  1:3.7 is a fairly typical differential gearing
+ *  0.15 is a magic constant based upon ~15" tyres
+ */
+static constexpr float wheel_diff = 3.7 * 0.15;
```

Comment:
Gear values are purposefully never displayed to the player. Naming constants and basing them on something near-reality is much better than the current system of arbitrary undocumented units.